packages/node_modules/@webex/plugin-meetings/src/meeting/index.js

```diff
@@ -318,6 +318,16 @@ const logRequest = (request, {header = '', success = '', failure = ''}) => {
  * @event meeting:self:lobbyWaiting
  * @instance
  * @type {Object}
+ * @property {Object} reason Reason why user left the meeting
```

Comment:
this doesn't look right - the reason should be in `meeting:self:left` event and not in `meeting:self:lobbyWaiting`

Reply:
yep typo

Reply:
looks like this PR got merged with this API typo not being fixed